File: server/bcephal-loader/src/main/java/com/moriset/bcephal/loader/domain/FileLoaderColumnDataBuilder.java
/**
 * 
 */
package com.moriset.bcephal.loader.domain;

import com.moriset.bcephal.utils.FileType;

import lombok.Data;

/**
 * @author Joseph Wambo
 *
 */
@Data
public class FileLoaderColumnDataBuilder {

	private int sheetIndex;

	private String sheetName;

	private String separator;

	private Long gridId;

	private FileType fileType;
	
	private Long templateId;

	private boolean hasHeader;

	private int headerRowcount;
}

File: client/bcephal-client-model/Loaders/FileLoaderColumnDataBuilder.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Bcephal.Models.Loaders
{
    public class FileLoaderColumnDataBuilder
    {

		public int SheetIndex { get; set; }

		public string SheetName { get; set; }

		public string Separator { get; set; }

		public long? GridId { get; set; }

		public FileType FileType { get; set; }

		public bool HasHeader { get; set; }

		public int HeaderRowcount { get; set; }


	}
}
